pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+markers =
+    redismod: run only the redis module tests
```

Comment:
I'd say this should be deselected by default, also I'd like to point out there is redis/redis-py#1715 in the parent repo.